content/200-concepts/200-database-connectors/07-mongodb.mdx

```diff
@@ -137,23 +137,33 @@ import { ObjectId } from 'bson'
 const id = new ObjectId()
 ```
 
-## Considerations
+## Limitations and known issues
```

Comment:
Huh, didn't I post a nitpicky comment here?

This wording "Limitations and known issues" feels more negative than "Considerations". These are not bugs or problems, these are just things different for MongoDB than for relational databases. 

So maybe "Differences to (Prisma for) relational databases"?

Reply:
I've renamed to 'Differences to connectors for relational databases', bit clunky but I agree the current title isn't right.